src/main/java/com/bootcamp/demo/model/Promotion.java

```diff
@@ -0,0 +1,37 @@
+package com.bootcamp.demo.model;
+
+public class Promotion {
+    public String getCode() {
+        return code;
+    }
+
+    private String code;
+    private int amount;
+    private boolean isFlatAmount;
+    private boolean enabled;
```

Comment:
Does it make sense to have an Enum here. Some possible values might be: ACTIVE, SUSPENDED, EXPIRED?